libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
Yeah, I agree. The only thing that wouldn't work is this:

```
define X
{
  rethrow
}

main
{
  install( default => X );
  throw( f )
}
```

which is fine but the semantic verifier would reject, I guess. Unless we're really really smart and we somehow check that when X is called it's always from inside of an install...! :-)